lua/fishtank/fish.lua

```diff
@@ -50,7 +48,7 @@ function Fish:initialize()
         zindex = 999,
         style = 'minimal',
         noautocmd = true,
-        border = '',
+        border = 'none',
```

Comment:
Just a drive-by warning fix - the lsp was moaning that '' wasn't allowed
```lua
--- @field border? 'none'|'single'|'double'|'rounded'|'solid'|'shadow'|string[]
```

Reply:
Huh, it just works now when I set up coc-lua. Not sure why I struggled with that in the past. Thanks!